File: TourApi/TourApi/Models/Hotel.cs
namespace TourApi.Models
{
    public class Hotel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int CityId { get; set; }
        public string ImageName { get; set; }
        public byte[] Image { get; set; }


        public string Transport {  get; set; }

        public int Price { get; set; }

        public bool IsActive { get; set; }

        public string Food { get; set; }
        public int Raiting { get; set; }
        
    }
}

File: my-app/src/app/Models/hotel.ts
export class Hotel{
    id:number;
    name:string;
    cityId:number;
    description: string;
    image:Blob;
    transport:string;
    price:number;
    isActive:boolean;
    food:string;
    raiting:number;

    constructor(id:number, name:string,cityId:number,image:Blob, transport:string, price:number,isActive:boolean,food:string,description: string, raiting:number){
        this.id=id;
        this.name=name;
        this.cityId=cityId;
        this.image = image;
        this.transport = transport;
        this.price = price;
        this.isActive = isActive;
        this.food = food;
        this.description = description;
        this.raiting = raiting;
    }
}